src/styles/Welcome.css

```diff
@@ -0,0 +1,30 @@
+.background {
+  background: url(/img/grocery-aisle.svg) no-repeat center center;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  width: 100vw;
+  height: 100vh;
+}
+
+.welcome-box {
+  background-color: #f9de59;
+  padding: 2rem;
+}
+
+.create-list-header {
+  margin-bottom: 2.75rem;
+}
+
+#welcome-header {
+  background-color: #ffffff;
+  margin: 1rem;
+  padding: 1.5rem;
+  border: lightgrey solid;
+  border-radius: 6px;
+}
+
+.create-button {
```

Comment:
I don't think we need this CSS rule. I've commented it out and there's no noticeable change

Reply:
@veeteeran did it not stretch the Create List button when you commented it out?

Reply:
@veeteeran did it not stretch the Create List button when you commented it out?